src/components/Button/theme.css

```diff
@@ -8,10 +9,8 @@
   border-radius: 0.3rem;
   background-color: var(--primary-medium);
   color: white;
-  text-transform: uppercase;
-  font-weight: normal;
+  font-family: var(--pullp-font);
```

Comment:
So I know this is annoying, but I would rename this variable, since Pullp has more than one font! Maybe `roboto-uppercase`?